homeassistant/components/device_sun_light_trigger.py

```diff
@@ -35,10 +48,10 @@ def setup(hass, config):
     light = get_component('light')
     sun = get_component('sun')
 
-    disable_turn_off = 'disable_turn_off' in config[DOMAIN]
+    disable_turn_off = CONF_DISABLE_TURN_OFF in config[DOMAIN]
     light_group = config[DOMAIN].get(CONF_LIGHT_GROUP,
                                      light.ENTITY_ID_ALL_LIGHTS)
-    light_profile = config[DOMAIN].get(CONF_LIGHT_PROFILE, LIGHT_PROFILE)
+    light_profile = config[DOMAIN].get(CONF_LIGHT_PROFILE)
     device_group = config[DOMAIN].get(CONF_DEVICE_GROUP,
                                       device_tracker.ENTITY_ID_ALL_DEVICES)
```

Comment:
I don't think that this would work because it looks like that the entities depends on the current user setup.